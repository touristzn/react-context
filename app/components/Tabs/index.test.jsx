import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { Tabs, Tab } from '.';

describe('<Tabs />', () => {
  const props = {
    activeIndex: 0,
    onTabChange: jest.fn(),
  };

  it('renders', () => {
    const renderer = new ShallowRenderer();
    const tree = 
      renderer.render(
        <Tabs {...props}>
          <Tab>1</Tab>
          <Tab>2</Tab>
        </Tabs>
      );
    expect(tree).toMatchSnapshot();
  });
});
