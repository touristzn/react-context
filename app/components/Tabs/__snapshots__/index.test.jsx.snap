// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tabs /> renders 1`] = `
<ul
  className="nav-tabs"
>
  <li
    className="tabs-item"
  >
    <a
      className="tab-link active"
      onClick={[Function]}
    >
      <Tab>
        1
      </Tab>
    </a>
  </li>
  <li
    className="tabs-item"
  >
    <a
      className="tab-link"
      onClick={[Function]}
    >
      <Tab>
        2
      </Tab>
    </a>
  </li>
</ul>
`;
